scalding-parquet-scrooge-fixtures/src/test/resources/test.thrift

```diff
@@ -86,14 +86,6 @@ struct StructWithReorderedOptionalFields {
   1: optional i32 fieldOne,
 }
 
-struct TestPersonWithRequiredPhone {
```

Comment:
In this file: https://github.com/twitter/scalding/blob/7ed0f92a946ad8407645695d3def62324f78ac41/scalding-parquet-scrooge-fixtures/src/test/resources/test.thrift it's on 89 and 107